docs/preview.md

```diff
@@ -10,6 +10,20 @@ Preview mode enables a collection of newer rules and fixes that are considered e
 Preview mode can be enabled with the `--preview` flag on the CLI or by setting `preview = true` in your Ruff
 configuration file (e.g. `pyproject.toml`).
 
+Preview mode can be configured separately for linting and formatting. To only enable preview lint rules without preview style formatting:
```

Comment:
Should we mention that this requires version 0.1.1+?

Reply:
Yeah I think so since we don't have versioned documentation yet.

Reply:
Done